src/components/Filter.tsx

```diff
@@ -1,9 +1,54 @@
-import React from "react";
-import { Select } from "@mui/material";
-import { myArray } from "../utils/constants";
+//IMPORT React and Child Components
+import * as React from "react";
+//IMPORT MUI packages
+import Select, { SelectChangeEvent } from "@mui/material/Select";
+import { Box, FormControl, InputLabel, MenuItem } from "@mui/material";
 
-function Filter() {
-  return <Select>{}</Select>;
+/*
+ * PARENT COMPONENT: FilterRows.tsx
+ * Props received from parent:
+ *   classificationLevel:
+ *      The taxon classification... kingdown, phylum etc
+ *   dropDownTaxons:
+ *      The taxon options corresponding to the classificationLevel
+ */
+interface filterProps {
+  classificationLevel: string;
+  dropDownTaxons: string[];
 }
 
+/*
+ * Main component Function.
+ * This component is a basic MUI select
+ * It receives an array of taxons for the user to select from
+ */
+const Filter = (props: filterProps) => {
+  const [taxon, setTaxon] = React.useState("");
+
+  const handleChange = (event: SelectChangeEvent) => {
+    setTaxon(event.target.value as string);
+  };
+
+  return (
+    <Box sx={{ minWidth: 120 }}>
```

Comment:
useStyles hook pls

Reply:
Removed style line for now. Will add again in next PR using style hook.